app/javascript/utils/index.tsx

```diff
@@ -1,6 +1,9 @@
 export const getUriWithParam = (baseUrl: string, urlParam: string, urlParamQuery: string): string => {
   const Url = new URL(baseUrl);
   const urlParams: URLSearchParams = new URLSearchParams(Url.search);
+  if (urlParam === 'query' && urlParams.has('page')) {
```

Comment:
This was added to handle a bug where you would not return to page 1 of results after entering a new search query.